SourcingTests/FetchedResultsDataProviderTests.swift

```diff
@@ -216,14 +216,25 @@ class FetchedResultsDataProviderTests: XCTestCase {
         XCTAssert(didUpdateDataSource)
     }
     
-    func testProcessUpdatesNumberForMoveChange() {
+    func testProcessUpdatesForMoveChange() {
         //Given
-        var updatesNumber = 0
+        var updateIndexPath = IndexPath()
+        var moveIndexPaths = [IndexPath]()
         
         dataProvider = try! FetchedResultsDataProvider(fetchedResultsController: fetchedResultsController, dataProviderDidUpdate: { _ in })
         dataProvider.whenDataProviderChanged = { updates in
-            if !updates!.isEmpty {
-                updatesNumber += 1
+            if updates!.isEmpty {
+                return
+            }
+                
+            for update in updates! {
+                switch update {
```

Comment:
This improves the tests. 

Could we add a test for the order of update calls. It is important that update happens after move

Reply:
ok - I will add this test today